README.md

```diff
@@ -1,5 +1,150 @@
+
+
+
+## INSTALL
+
+Installing **ManimPango** is super easy, just use pip to install it. It is [`manimpango`](https://pypi.org/project/manimpango/) in PyPi.
+
+```sh
+pip install manimpango
+```
+
+#### Note:
+
+For **Linux Users**, installing the binary wheel provided in PyPi could cause some weird things happen to your system.
+
+For example, you would get a folder named `fontconfig`, in your directory, and it will have a lot of cache files each time you run Manim. To avoid those kind of weird things from happening, you would have to install it from source using the below command.
+
+```sh
+pip install manimpango --no-binary :all:
+```
+
+For installing that this way you would need a C compiler as well as Pango and its dependencies along with it's Development headers. See [BUILDING](#BUILDING ) for more information.
+
+## BUILDING
+
+### Linux/MacOS
+
+For building **ManimPango**, you would need a C compiler,python's development headers,[`pkg-config`](https://www.freedesktop.org/wiki/Software/pkg-config/),[Pango](https://pango.gnome.org) along with its development headers and its dependencies.
+
+If you are on macOS, you can use [brew](https://brew.sh) to install those
+
+```sh
+brew install pango pkg-config
+```
+
+If you are on Linux, you can use a system package manager to do so. For example, if you are on Debian based system, you can use `apt`
+
+```sh
+apt install libpango1.0-dev pkg-config python3-dev
+```
+
+**Arch Linux:** `pacman -S pango pkgconf`
+
+**Fedora:** `dnf install pango-devel pkg-config python3-devel`
+
+Or similar in your system's package manager.
+
+##### Using `tar` archives
+
+If you don't want to contribute to this Repository, you can use the tar archives published in PyPi, or just use pip to install using
+
+```sh
+pip install manimpango --no-binary :all:
+```
+
+**Note**: `pip` by default uses wheels, so make sure to pass `--no-binary` parameter.
+
+##### Using `git` clones
+
+If you are using a clone of this repository, then you would need [Cython](https://cython.org) which can be easily installed using pip
+
+```sh
+pip install Cython
+```
+
+After that you can use pip to install the clone using
+
+```sh
+pip install .
+```
+
+You would need to this way if you want to *Contribute* to **ManimPango**.
+
+You should do these inside your poetry shell, if you want to use it with **Manim**.
+
+### Windows
+
+*If you are a normal user, don't read this, you have wheels which you can just install directly using pip.*
+
+If you want to contribute to **ManimPango** and you are on Windows, this section is for you.
+
+First you would need a C compiler, as windows doesn't have one by default, you have two choices
+
+1. MinGW/Msys2
+
+2. Visual Studio
+
+#### MinGW/Msys2
+
+1. Download Msys2 from as specified in their page https://www.msys2.org/#installation
+2. Open, a `MSYS2 MINGW64 shell`, and not anything else. Small hint, it has a blue color logo.
```

Comment:
This is not quite clear to me. They have to **install** Msys2 first, following the installation instructions on the linked page. And then they have to open that shell? Is this the `mintty`?

Reply:
Installing MSYS2 will provide 3 different types of C compiler and each can be opened in different shells. Here `MINGW64` means 64 bit shell. Either MING64 or MINGW32 must be used or else things won't work.
See https://wiki.inkscape.org/wiki/index.php?title=Compiling_Inkscape_on_Windows_with_MSYS2
Which is mintty of course, they would have to search for `MSYS2 MINGW64 shell` in the start menu and open it. Its logo is blue coloured M.

Reply:
Also, tell me how to make it clearer?

Reply:
```suggestion
2. Once you have **MSYS2** installed,  it offers you three different shells: the **MINGW32** shell, the **MINGW64** shell and **MSYS** shell. In order for the following steps to work, you have to open the **MSYS2 MINGW64** shell (you can search for this). Small hint: it has a blue color logo.
```